.circleci/config.yml

```diff
@@ -32,7 +32,11 @@ jobs:
           environment:
           command: |
             export PATH=$HOME/conda/envs/glow/bin:$PATH
-            sbt printTests coverage "testOnly $(circleci tests split --split-by=timings --timings-type=classname core-test-names.log | tr '\n' ' ') --" coverageReport exit
+            if [[ $CIRCLE_NODE_INDEX == 0 ]]; then
```

Comment:
What's up with this?

Reply:
I'm running the 2.11 tests on the first node, and 2.12 on the second.

Reply:
Is it common to run tests against both Scala versions on every PR? What does Spark do?

Reply:
For each PR on 2.4, it looks like they run the tests on the matrix of Scala 2.11/2.12 and Hadoop 2.6/2.7.